### PR TITLE
Post: Adds the post on Ansible & macOS.

### PR DESCRIPTION
**Pull request by nickcharlton**

### CODE REVIEW

**Comment by danielgroves** [comment]
This reads well. Clear and concise. 

My one criticism is you don't make it easy for me to apply this to my own scenario: i.e. where do you find out what options are available via the command line? Would be good to see a link out to something that documents them (officially, or otherwise).

**Comment by nickcharlton** [comment]
> i.e. where do you find out what options are available via the command line?

**Comment by danielgroves** [comment]
Yeah, pretty much

File: _posts/2016-11-25-automating-macos-using-ansible.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+---
+
+I have a collection of macOS machines, some of which are used for services like
+[Jenkins][] and others which are used for testing. These all have a consistent
+base configuration which I've been using Ansible to configure.
+
+Ansible is being used to to drive a set of utilities that have emerged to help
+with some of the rougher edges of configuring macOS. In places Where this
+abstraction isn't available, I'm calling out to `defaults write` to set
+settings directly. There's a [few][my_dotfiles] [examples][] and a [site
```
Comment (danielgroves) [inline _posts/2016-11-25-automating-macos-using-ansible.md]: `[examples][]` needs to link somewhere

File: _posts/2016-11-25-automating-macos-using-ansible.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+---
+- name: Base OS X configuration
+  hosts: all
+  roles:
+    - geerlingguy.homebrew
+    - base
+```
+
+This is the base playbook. It's held in a central `ansible` repo, but here
+we're installing Homebrew (using [Jeff Geerling's Homebrew Role][hb_role] and
```
Comment (danielgroves) [inline _posts/2016-11-25-automating-macos-using-ansible.md]: You open a bracket here, but never close it

File: _posts/2016-11-25-automating-macos-using-ansible.md (hunk 1 above)
Comment (nickcharlton) [inline _posts/2016-11-25-automating-macos-using-ansible.md]: Oh, that should've been: `bynens_dotfiles`.

**Pull request merged**